examples/1_feature_overview/example.py

```diff
@@ -400,15 +401,15 @@ def f(x):
 
 
 # Without thresholds matrix
-sr3_optimizer = ps.SR3(threshold=0.1, thresholder="l0")
-model = ps.SINDy(optimizer=sr3_optimizer).fit(x_train, t=dt)
+weight = ps.SR3.calculate_l0_weight(0.1, 1)
+sr3_optimizer = ps.SR3(reg_weight=0.1, regularizer="l0")
 print("Threshold = 0.1 for all terms")
 model.print()
 
 # With thresholds matrix
-thresholds = 2 * np.ones((3, 10))
-thresholds[4:, :] = 0.1
-sr3_optimizer = ps.SR3(thresholder="weighted_l0", thresholds=thresholds)
+weights = 2 * np.ones((3, 10))
+weights[4:, :] = ps.SR3.calculate_l0_weight(0.1, 1)
+sr3_optimizer = ps.SR3(regularizer="weighted_l0", reg_weight=weights)
```

Comment:
Shouldn't the rest of `weights` get the same treatment as `weights[4:, :]`?

Reply:
The current example only uses a 0.1 threshold on the 4th column so I simply translate it to weight.